mk/cert-manager.mk

```diff
@@ -10,3 +10,10 @@ cert-manager-install:  ## Install cert-manager operator
 cert-manager-uninstall:  ## Delete cert-manager operator
 	kubectl delete -f config/certmanager/subscription.yaml --wait=true
 
+.PHONY: cert-manager-self-signed-issuer-create
+cert-manager-self-signed-issuer-create:  ## Create a cluster self signed issuer
+	kubectl create -f config/certmanager/clusterissuer-selfsigned.yaml
```

Comment:
Since we're targeting OpenShift platform and not a vanilla kubernetes distribution (we depend on routes, SCCs and maybe some other OpenShift-specific resources), I think we should stick with `oc` and use it everywhere.

Reply:
Updated! thanks!